src/global/Colors.js

```diff
@@ -11,6 +11,7 @@ getBitNationLightBlue = (alpha) => {
 
 module.exports = {
 
+  // Why do we have these definitions?
```

Comment:
Good question. If they are not used remove them. All of the unused once.

Reply:
Cannot remove until we confirm no one is using them. They cause no harm; let us leave them for now.